webservices/resources/filings.py

```diff
@@ -61,7 +61,10 @@ def args(self):
 
     def get(self, **kwargs):
         if kwargs.get('form_type') and 'RFAI' in kwargs.get('form_type'):
-            kwargs['form_type'] = ['RFAI', 'FRQ']
+            #If user happened to also pass in 'FRQ', remove it to prevent unnecessary filtering
+            #by appending to the list we don't wipe out the other form_type(s) passed in
+            kwargs.get('form_type').remove('FRQ')
```

Comment:
If somebody is explicitly looking for `FRQ` files this will not alter that behavior.